src/app/window.cpp

```diff
@@ -94,6 +100,11 @@ QTabWidget *MainWindow::tabs_widget()
     media->fill_tabs();
     settings->fill_tabs();
 
+	CameraSettingsSubTab* cameraSettings = settings->findChild<CameraSettingsSubTab*>();
+	connect(cameraSettings, &CameraSettingsSubTab::cam_name_changed, camera, &CameraTab::on_camName_changed);
+	connect(cameraSettings, &CameraSettingsSubTab::cam_toggle_requested, camera, &CameraTab::on_toggle_connect);
+	connect(camera, &CameraTab::media_status_changed, cameraSettings, &CameraSettingsSubTab::on_newConnectionStatus);
```

Comment:
I bypassed the whole Config object because I felt settings could be handled simply without it. Happy to discuss.